DEVELOPERS_GUIDE.md

```diff
@@ -154,13 +156,32 @@ Next we need to deploy firebase rules so not everyone can just go ahead and writ
 $ npx firebase login
 ```
 
-Once logged in, we can deploy the provided rules by executing:
+Once logged in, make sure you update the local configuration file under `firebase/functions/.firebaserc` to point to your own firebase project:
+
+```
+{
+  "projects": {
+    "default": "<PRODUCTION PROJECT_ID>",
```

Comment:
Should that rather be the dev project id? Because that's what you usually develop on.

Reply:
sure. my bad.